setup.py

```diff
@@ -51,7 +51,7 @@ def get_version_and_cmdclass(pkg_path):
 
 # Any additional requirements besides OctoPrint should be listed here
 # todo after dropping 3.7 remove typing-extensions (used by model.py and listing.py)
-plugin_requires = ["RPi.GPIO", "typing-extensions"]
+plugin_requires = ["gpiozero", "typing-extensions", "lgpio"]
```

Comment:
Thank you @ch3p4ll3 !
Could you please clarify "lgpio": what's the purpose of it — I didn't find its usage in your changes.

Reply:
Hi, lgpio is used by gpiozero for building the [pin factory.](https://gpiozero.readthedocs.io/en/v1.6.0/api_pins.html) so there is no "direct" usage of it

Reply:
So, `gpiozero` automatically detects that `lgpio` is installed and then use it as a pin factory, is that correct @ch3p4ll3 ?

Reply:
Yep, exactly